pkg/controllers/updaterun/initialization.go

```diff
@@ -250,6 +250,12 @@ func (r *Reconciler) generateStagesByStrategy(
 		// other err can be retried.
 		return controller.NewAPIServerError(true, err)
 	}
+	// Remove waitTime from the after stage tasks of type Approval.
+	if err := r.removeWaitTimeFromAfterStageTasks(ctx, &updateStrategy); err != nil {
+		klog.ErrorS(err, "Failed to remove wait time from after stage tasks", "clusterStagedUpdateStrategy", updateStrategy.Name, "clusterStagedUpdateRun", updateRunRef)
+		return fmt.Errorf("%w: %s", errInitializedFailed, err.Error())
```

Comment:
This error is from apiserver and can be retried, so it should not be wrapped with `errInitializedFailed`. See an example here: https://github.com/kubefleet-dev/kubefleet/blob/main/pkg/controllers/updaterun/initialization.go#L315C4-L316C50